wsgi.py

```diff
@@ -32,24 +28,72 @@ def wake_up():
     ai_service_id = input_data.get('ai_service', 'generic_ai')
     raw_data = input_data['data']
 
-    s3_destination = f'{AWS_BUCKET}/{data_id}/{ai_service_id}'
-
-    application.logger.info(
-        'Saving data to location:  s3://%s', s3_destination
-    )
-
-    filesystem = s3.connect(AWS_KEY, AWS_SECRET)
-    s3.save_data(filesystem, s3_destination, raw_data)
-
-    message = {
-        'message': f'AI-Ops pipeline successfull for {data_id}',
-        'url': s3_destination
+    try:
+        temp_file_name = tempfile.NamedTemporaryFile(delete=False).name
+        with tarfile.open(temp_file_name, "w:gz") as tar:
+            data = io.BytesIO(json.dumps(raw_data).encode())
+            info = tarfile.TarInfo(name=f'{ai_service_id}_{data_id}.json')
+            info.size = len(data.getvalue())
+            temp_file_name = tar.name
+            tar.addfile(info, data)
+
+    except (IOError, TypeError, tarfile.TarError) as e:
```

Comment:
`TypeError` - why is this being risen/caught? I know you've already said you've seen it, but I'm not sure what was the cause. The only case I can think of is when you're trying to serialize something which the `json.dump` is not able to comprehend. That case should have a default fallback. si there any other reason to suppress this exception?

Reply:
Addressed in the last commit